Technical-Steering-Committee/principles.md

```diff
@@ -0,0 +1,28 @@
+# Principles
+
+Set of principles that Zowe agrees on following. In case of disagreement within the Zowe, the principles outlined here needs to be taken into account.
+
+What Zowe is looking for? High-quality, high-velocity projects that fit 'mainframe' and link it to the distributed environments allowing for better interaction between applications developed and operated in hybrid environments. 
+
+Our role as Zowe towards the squads is first to do No harm. 
+
+## Zowe keeps the zOS security Standards
+**Principle:** Secure
+
+## zOS is a valid player within development ecosystem
+**Principle:** Keep zOS workloads on zOS
+
+## The Zowe doesn't depend on and isn't owned by any specific Vendor
+**Principle:** Vendor Neutral Development
+
+## Squads are self-governing
+**Principle:** Minimal Viable Governance
+
+The Zowe and TSC pushes only for the minimum necessary governance required for the ZOwe project to be overall dependable and successful. 
+
+## Not a Standards Body
+**Principle:** Zowe promotes interoperability via interfaces that get real-world use
```

Comment:
We don't want to be unknown technology. We want to introduce the management to the MainFrame that is common / usable across the platforms.

Reply:
Embrace existing standards as well as bringing standards from the other platforms to the mainframe.THe goal is standard

Reply:
Embrace ... while introducing trending capabilities.

Reply:
Bringing interactions to the Main Frame